### PR TITLE
DO NOT APPROVE THIS PR

### PR DESCRIPTION
**Pull request by npmccallum**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # enarxbot
 I'm a bot. Bleep. Blorp.
+
```
Comment (mbestavros) [inline README.md:3]: Testing 1, 2, 3

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
+!!!DO NOT APPROVE THIS PR!!!!
```
Comment (connorkuehl) [inline README.md:4]: blorgle

File: README.md (hunk 1 above)
Comment (ambaxter) [inline README.md:3]: This also needs changes

File: README.md [hunk 3]
```diff
@@ -1,2 +1,4 @@
 # enarxbot
 I'm a bot. Bleep. Blorp.
```
Comment (ambaxter) [inline README.md:2]: You are not.

**Comment by npmccallum** [comment]
Thanks everyone! Closing.

**Pull request closed without merging**